src/screens/ContactScreen/index.js

```diff
@@ -129,14 +192,15 @@ const ContactScreen = ({navigation}) => {
       username={item.username}
       followed={extendedState.followed}
       userid={item.user_id}
+      isAnon={isAnon}
       onPress={() => handleSelected(item)}
     />
   );
   // }
   const handleSelected = (value) => {
-    const copyFollowed = [...followed];
-    const copyUsername = [...usernames];
-    const copyUsers = [...selectedUsers];
+    const copyFollowed = isAnon ? [] : [...followed];
+    const copyUsername = isAnon ? [] : [...usernames];
+    const copyUsers = isAnon ? [] : [...selectedUsers];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ItemUser` component now receives an additional prop `isAnon`. Also, the `handleSelected` function has been updated to handle anonymous chats. The `isAnon` check is used to decide whether to create an anonymous chat or a signed chat. This is a good practice as it allows the function to behave differently based on the context it is used in.
```diff
+ isAnon={isAnon}
+ onPress={() => handleSelected(item)}
+ const copyFollowed = isAnon ? [] : [...followed];
+ const copyUsername = isAnon ? [] : [...usernames];
+ const copyUsers = isAnon ? [] : [...selectedUsers];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->